### PR TITLE
feat: [CI-11942]: icon for CI intelligent test step

### PR DESCRIPTION
**Pull request by shaurya-harness**
icon for CI intelligent test step
 
 
 You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- StyleLint: `retrigger stylelint`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- ImageSnapshot `retrigger ImageSnapshot`

### CODE REVIEW

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2411

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2413

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2417

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2419

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2420

File: packages/icons/src/ci-test-step.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....823 13.734a.5.5 0 0 1 .5-.5h6.585a.5.5 0 0 1 .5.5v3.533a.5.5 0 0 1-.5.5H2.336a.5.5 0 0 1 0-1h8.072v-2.533H4.323a.5.5 0 0 1-.5-.5ZM23.733 3.104a.5.5 0 0 1 0 .707l-2.366 2.366a.5.5 0 0 1-.354.147h-8.808a.5.5 0 0 1 0-1h8.601l2.22-2.22a.5.5 0 0 1 .707 0ZM12.678 9.752a.5.5 0 0 1 .5-.5h7.835a.5.5 0 0 1 0 1h-7.835a.5.5 0 0 1-.5-.5ZM14.096 12.539a.5.5 0 0 1 .673-.218l3.624 1.85h6.17a.5.5 0 1 1 0 1h-6.29a.5.5 0 0 1-.228-.054l-3.731-1.906a.5.5 0 0 1-.218-.672ZM16.767 17.62a.5.5 0 0 1 .5-.5h3.352a.5.5 0 0 1 .366.158l2.76 2.957a.5.5 0 0 1-.731.683l-2.612-2.799h-3.135a.5.5 0 0 1-.5-.5Z" clip-rule="evenodd"/><path fill="#4F5162" fill-rule="evenodd" d="M24.755 1a1.077 1.077 0 1 0 0 2.155 1.077 1.077 0 0 0 0-2.155Zm-2.078 1.077a2.077 2.077 0 1 1 4.155 0 2.077 2.077 0 0 1-4.155 0ZM22.788 8.872a1.077 1.077 0 1 0 0 2.155 1.077 1.077 0 0 0 0-2.155ZM20.71 9.95a2.077 2.077 0 1 1 4.154 0 2.077 2.077 0 0 1-4.154 0ZM26.337 13.594a1.077 1.077 0 1 0 0 2.155 1.077 1.077 0 0 0 0-2.155Zm-2.077 1.078a2.077 2.077 0 1 1 4.154 0 2.077 2.077 0 0 1-4.154 0ZM24.365 21.077a1.077 1.077 0 1 0 0 2.154 1.077 1.077 0 0 0 0-2.154Zm-2.077 1.077a2.077 2.077 0 1 1 4.155 0 2.077 2.077 0 0 1-4.155 0Z" clip-rule="evenodd"/></svg>
```
[the remark this answers is not in the document]
Comment (shaurya-harness) [reply packages/icons/src/ci-test-step.svg]: updated

**Review by vardanbansal-harness — APPROVED** [review]

**Comment by bot2-harness** [comment]
Image snapshots failed please check https://console.cloud.google.com/storage/browser/ui-team-bucket/uicore-2421

**Review by karansaraswat19 — APPROVED** [review]

**Pull request merged**